go.mod

```diff
@@ -6,8 +6,7 @@ require (
 	code.gitea.io/sdk/gitea v0.15.1
 	github.com/golang/mock v1.6.0
 	github.com/kelseyhightower/envconfig v1.4.0
-	github.com/keptn/go-utils v0.16.0
-	github.com/keptn/kubernetes-utils v0.14.0
+	github.com/keptn/go-utils v0.17.0
```

Comment:
The `GetClientSet` function is only available in the newest version, so @christian-kreuzberger-dtx and I decided to use it instead of reimplementing the functionality.